src/ColorSpace.d.ts

```diff
@@ -5,6 +5,8 @@
  */
 import { White } from "./adapt.js";
 import { ColorConstructor, Coords, ColorTypes } from "./color.js";
+import type FormatClasa from "./Format.js";
```

Comment:
```suggestion
import type FormatClass from "./Format.js";
```

Reply:
Whoops. Other references to this typo in the file will also need to be changed

Reply:
Nevermind—it looks like the typo is only in the import